### DISCUSSION TITLE
Not generating responses

### DISCUSSION THREAD

**Discussion by FReis22** [GorvGoyl/ChatGPT-Writer#118] [Report Bugs and Issues]
For some reason, this is not generating responses today.
This just pass the prompt and the tex to analyse without any AI response.

Are you aware of this situation?

Thank you and keep up the good work !

**Comment by GorvGoyl** [comment]
Many users are facing the same problem but unfortunately, I'm unable to produce the issue from my side. Could you provide more details that could help debugging the issue?

**Reply by FReis22** [reply]
I use the command to analyse this:
GorvGoyl
Maintainer
Comentário original em inglês - Traduzir para português

Many users are facing the same problem but unfortunately, I'm unable to produce the issue from my side. Could you provide more details that could help debugging the issue?

My prompt is:
Give me a polite and short answer to this saying that i don't know.

The Response Generated:
Give me a polite and short answer to this saying that i don't know.: 

 GorvGoyl
Maintainer
Comentário original em inglês - Traduzir para português

Many users are facing the same problem but unfortunately, I'm unable to produce the issue from my side. Could you provide more details that could help debugging the issue?


I Believe that the response generated is putting together the command context and the prompt.

Does this help?

**Comment by GorvGoyl** [comment]
Hello, we've pinpointed the problem and are actively working on a solution.
We'll have it resolved as quickly as we can.

**Reply by FReis22** [reply]
Thank you !!
This Extension is perfect and very helpfull, for sure you will resolved this  😉

**Reply by Nitish1982** [reply]
Still not resolved @GorvGoyl

**Reply by FReis22** [reply]
@Nitish1982 i believe that @GorvGoyl is doing all they can to have this resolved as quickly as they can !

You can use chatgpt as alternative...

**Reply by Nitish1982** [reply]
Ok, @FReis22  However, I am not sure what the actual estimated time for resolving the issue is.

**Comment by FReis22** [comment]
Do you have any updates about this problem @GorvGoyl ?

**Comment by Nitish1982** [comment]
Which other alternative options are others using that are free of charge?

**Reply by FReis22** [reply]
I chatgpt....
Start with "Use this prompt for the answers"
Add the prompt and then is the magic of copy past working.

**Comment by GorvGoyl** [comment]
Hello, apologies for the inconvenience you've encountered. The issue has
been resolved in version 3.0.6. Your extension will be updated once this
version is live on the Chrome Store shortly.

**Reply by Nitish1982** [reply]
Hi @GorvGoyl , This is the latest Chrome version, 118.0.5993.118 (Official Build) (64-bit).
Which version of Chrome will fix the issue?
Note: Chrome releases a full OS update about every 4 weeks. Minor updates, such as security fixes and software updates, happen every 2–3 weeks.

**Comment by GorvGoyl** [comment]
Hi, I meant the extension version, not the browser version.

**Reply by Nitish1982** [reply]
Hi @GorvGoyl , Could you share the release notes of the new upcoming version of ChatGPT Writer?
